### ISSUE TITLE
dnorm.jl could have a simplier objective function

### ISSUE THREAD

**Issue by aoboturov** [BBN-Q/SchattenNorms.jl#3]
It looks like that instead of:

```
prob = maximize( retrϕ( ϕ(Jr,Ji)'*ϕ(Xr,Xi) ) )
```

you could just write:

```
prob = maximize( trace( Jr*Xr ) )
```

why? check `eqn 1` from [Approximation algorithms for Max-3-Cut and other problems via complex semidefinite programming](http://www.sciencedirect.com/science/article/pii/S0022000003001454) which could also be applied here.

**Comment by marcusps** [comment]
Thanks for the reference! I have been looking for a good reference on the equivalence of complex and real semidefinite programs (for various other reasons). I'll check it out and make the appropriate changes.

**Comment by aoboturov** [comment]
I actually thought to work on that problem in relation to Convex.jl
project. I am interested to contribute to the solver interface. Could I
contact you somehow by email?

**Comment by aoboturov** [comment]
[proposal.pdf](https://github.com/BBN-Q/SchattenNorms.jl/files/263775/proposal.pdf)

I thought to do some work on it. Here's my proposal attached as a pdf.

**Comment by marcusps** [comment]
You can e-mail me [here](http://www.google.com/recaptcha/mailhide/d?k=01I9CD1FZxihXzC2jDYjxA4w==&c=ORGeAWPfsQNZ5r3iqTLy-Dzo0OO9d4iYMU_z7C0wl2Q=)

**Comment by aoboturov** [comment]
Actually, I got the formula wrong initially. the correct one looks like:

```
prob = maximize( trace( Jr*Xr + Ji*Xi ) )
```

Fixed in #4
